tests/unit_tests/docmaps/v2/codecs/docmaps_test.py

```diff
@@ -297,8 +298,11 @@ def test_should_not_populate_actions_peer_reviewed_step_if_tags_are_empty(self):
         )
         peer_reviewed_step = docmaps_item['steps']['_:b1']
         peer_reviewed_actions = peer_reviewed_step['actions']
-        assert len(peer_reviewed_actions) == 0
-        assert peer_reviewed_actions == []
+        assert len(peer_reviewed_actions) == 1  # 1 rp_meca_path content
+        assert peer_reviewed_actions[0] == get_rp_meca_path_action(
```

Comment:
Just wondering whether comparing the list directly would be helpful (not checking the length):

```python
        assert peer_reviewed_actions == [get_rp_meca_path_action(
            query_result_item=DOCMAPS_QUERY_RESULT_ITEM_WITH_EVALUATIONS,
            manuscript_version=MANUSCRIPT_VERSION_1
        )]
```